LukeClient/luke-client.py

```diff
@@ -32,9 +37,47 @@
                 'Ram': server.serverRam.ramObject,
                 'NICs': server.serverNics.nicsObject,
                 'Disks': server.serverDisks.disksObject}
+port = 8000
+
+# report = convert_to_json(serverObject)
+report = "{\"Vendor\": \"vend\"," \
+                     " \"Cpu\": {\"Sockets\": \"1\", \"Arch\": \"x86_64\", \
+                     \"Speed\": \"2201.000\", \"Cores\": \"1\"}," \
+                     " \"Ram\": {\"Size\": \"3062784\"}, " \
+                     "\"NICs\": {\"ens33\": " \
+                     "{\"Speed\": \"1000Mb/s\", \"Mac\": \"00:0c:29:3d:5e:ce\", \
+                     \"Type\": \"Twisted Pair\", \"ip\": \"192.168.0.4\"}}," \
+                     " \"Disks\": {\"sda\": {\"Vendor\": \"VMware\", \"Size\": \"2\"}, " \
+                     "\"sr0\": {\"Vendor\": \"VMware\", \"Size\": \"5\"}}, " \
+                     "\"ip\": \"192.168.0.5\", \"Model\": \"mod\"}"
+
+req = "{\"requirements\": {\"Cpu\": {\"Sockets\": \"1\",\
+                \"Speed\": \"2201.000\", \"Cores\": \"1\"}," \
+              "\"Vendor\": \"vend\"}," \
+              "\"other_prop\": {\"Ram\": {\"Size\": \"3062784\"}, " \
+              "\"NICs\": {\"ens33\": " \
+              "{\"Speed\": \"1000Mb/s\", \"Mac\": \"00:0c:29:3d:5e:ce\", \
+              \"Type\": \"Twisted Pair\"}}," \
+              " \"Disks\": {\"sda\": {\"Vendor\": \"VMware\", \"Size\": \"2\"}, " \
+              "\"sr0\": {\"Vendor\": \"VMware\", \"Size\": \"5\"}}, " \
+              "\"Model\": \"mod\", \"profile\": \"common\"}}"
+
+
+request = post("http://localhost:{}/request/".format(port), data={"request": req})
```

Comment:
I don't think that luke-client should make requests... it's just for testing purposes, right?